### PR TITLE
Add health check for the cluster CPUs

### PR DESCRIPTION
**Pull request by sjparkinson**
Here's what I ended up with, it's a bit hard coded (all the information should be able to be pulled out of the environment) but would like any feedback for this first pass.

There are errors with the IAM role:

```
Failed to get CloudWatch data error_message="User: arn:aws:iam::027104099916:user/FTApplicationUserFor_next-es-interface-write.dev is not authorized to perform: cloudwatch:GetMetricStatistics" error_name=AccessDenied error_stack="AccessDenied: User: arn:aws:iam::027104099916:user/FTApplicationUserFor_next-es-interface-write.dev is not authorized to perform: cloudwatch:GetMetricStatistics;     at Request.extractError (/home/samuel/Repositories/github.com/Financial-Times/next-es-interface/node_modules/aws-sdk/lib/protocol/query.js:40:29);     at Request.callListeners (/home/samuel/Repositories/github.com/Financial-Times/next-es-interface/node_modules/aws-sdk/lib/sequential_executor.js:105:20);     at Request.emit (/home/samuel/Repositories/github.com/Financial-Times/next-es-interface/node_modules/aws-sdk/lib/sequential_executor.js:77:10);     at Request.emit (/home/samuel/Repositories/github.com/Financial-Times/next-es-interface/node_modules/aws-sdk/lib/request.js:668:14);     at Request.transition (/home/samuel/Repositories/github.com/Financial-Times/next-es-interface/node_modules/aws-sdk/lib/request.js:22:10);     at AcceptorStateMachine.runTo (/home/samuel/Repositories/github.com/Financial-Times/next-es-interface/node_modules/aws-sdk/lib/state_machine.js:14:12);     at /home/samuel/Repositories/github.com/Financial-Times/next-es-interface/node_modules/aws-sdk/lib/state_machine.js:26:10;     at Request.<anonymous> (/home/samuel/Repositories/github.com/Financial-Times/next-es-interface/node_modules/aws-sdk/lib/request.js:38:9);     at Request.<anonymous> (/home/samuel/Repositories/github.com/Financial-Times/next-es-interface/node_modules/aws-sdk/lib/request.js:670:12);     at Request.callListeners (/home/samuel/Repositories/github.com/Financial-Times/next-es-interface/node_modules/aws-sdk/lib/sequential_executor.js:115:18);     at Request.emit (/home/samuel/Repositories/github.com/Financial-Times/next-es-interface/node_modules/aws-sdk/lib/sequential_executor.js:77:10);     at Request.emit (/home/samuel/Repositories/github.com/Financial-Times/next-es-interface/node_modules/aws-sdk/lib/request.js:668:14);     at Request.transition (/home/samuel/Repositories/github.com/Financial-Times/next-es-interface/node_modules/aws-sdk/lib/request.js:22:10);     at AcceptorStateMachine.runTo (/home/samuel/Repositories/github.com/Financial-Times/next-es-interface/node_modules/aws-sdk/lib/state_machine.js:14:12);     at /home/samuel/Repositories/github.com/Financial-Times/next-es-interface/node_modules/aws-sdk/lib/state_machine.js:26:10;     at Request.<anonymous> (/home/samuel/Repositories/github.com/Financial-Times/next-es-interface/node_modules/aws-sdk/lib/request.js:38:9);     at Request.<anonymous> (/home/samuel/Repositories/github.com/Financial-Times/next-es-interface/node_modules/aws-sdk/lib/request.js:670:12);     at Request.callListeners (/home/samuel/Repositories/github.com/Financial-Times/next-es-interface/node_modules/aws-sdk/lib/sequential_executor.js:115:18);     at callNextListener (/home/samuel/Repositories/github.com/Financial-Times/next-es-interface/node_modules/aws-sdk/lib/sequential_executor.js:95:12);     at IncomingMessage.onEnd (/home/samuel/Repositories/github.com/Financial-Times/next-es-interface/node_modules/aws-sdk/lib/event_listeners.js:211:11);     at emitNone (events.js:91:20);     at IncomingMessage.emit (events.js:185:7);     at endReadableNT (_stream_readable.js:974:12);     at _combinedTickCallback (internal/process/next_tick.js:74:11);     at process._tickDomainCallback (internal/process/next_tick.js:122:9)" level=error
```

I'll look into what policy changes need to be made.

### CODE REVIEW

**Comment by sjparkinson** [comment]
The write users have now been added to a group with access to `cloudwatch:GetMetricStatistics`.

**Comment by sjparkinson** [comment]
There's plenty more checks we can add too, see https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/es-metricscollected.html. Figure it's good to get something out the door first though.

File: server/health-checks/elasticsearch-cluster.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+			cloudWatchStatistic: 'Average',
+			cloudWatchDimensions: [
+				{ Name: 'DomainName', Value: 'nextcontent' },
+				{ Name: 'ClientId', Value: '027104099916' }
+			],
+			samplePeriod: 300,
+			threshold: 60
+		},
+		{
+			name: 'US data instances CPU utilization above 60% threshold',
```
Comment (i-like-robots) [inline server/health-checks/elasticsearch-cluster.js]: I wonder if we can make this healthcheck region specific (available from the environment, `process.env.REGION`) so we don't set off multiple alarms

**Comment by i-like-robots** [comment]
LGTM (pending rebase)

**Comment by sjparkinson** [comment]
Awesome, thanks for looking, will merge it later today.

**Pull request merged**